.github/workflows/go.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [1.13, 1.14, 1.15, 1.16, 1.17]
+        go-version: [1.15, 1.16, 1.17]
```

Comment:
The newly fixed jwt-go does not support versions prior to go 1.5?

Reply:
No. They do not support. Here are two issues explaining why. I posed in the original motivation too.
golang-jwt/jwt#50
golang-jwt/jwt#90